cmd/config-translator/translator.go

```diff
@@ -32,6 +32,8 @@ func initFlags() {
 	var inputMode = flag.String("mode", "ec2", "Please provide the mode, i.e. ec2, onPrem")
 	var inputConfig = flag.String("config", "", "Please provide the common-config file")
 	var multiConfig = flag.String("multi-config", "remove", "valid values: default, append, remove")
+	var strictValidation = flag.Bool("strict-validation", false, "Return exit code 98 if validating agent's json config fails and 99 if no config json was found.")
```

Comment:
For the default behavior, the translator will have a non-exit zero code when **--multi-config is default or append** even if the translator is having an invalid config or having no json file since it would use a default JSON config for that particular OS. So the cx have get used to it for a long time; therefore, I honest don't want to change this behavior for them though.